### PR TITLE
comment unused dependency

### PR DESCRIPTION
**Pull request by atefBB**
Deprecations: yes.
This dependency should be removed (I think).

### CODE REVIEW

**Comment by Nyholm** [comment]
Hey. 
Thank you for this PR. The import is used here in the comment: https://github.com/php-http/httplug/pull/136/files#diff-add00846710b59ac0af2a07e1ede2747R22

**Comment by atefBB** [comment]
Hey @Nyholm!
What about this: `@return Psr\Http\Message\ResponseInterface`

**Comment by GrahamCampbell** [comment]
> What about this: @return Psr\Http\Message\ResponseInterface

Actually, you'll need a leading slash on that, in order for it to resolve from the global namespace.

**Comment by Nyholm** [comment]
Im not sure what we win by this change... Why is one better than the other?

File: src/HttpClient.php [hunk 1]
```diff
@@ -19,7 +18,7 @@
      *
      * @param RequestInterface $request
      *
-     * @return ResponseInterface
+     * @return Psr\Http\Message\ResponseInterface
```
Comment (dbu) [inline src/HttpClient.php:21]: a return type annotation is also a dependency on the class from the other namespace. imho we should use `use` statements for those as well.

**Comment by sagikazarmark** [comment]
I agree with the others, not sure what's the advantage, I'm happy with the current situation. It's a trivial thing anyway. Thanks for the contribution though.

File: src/HttpClient.php (hunk 1 above)
Comment (atefBB) [reply src/HttpClient.php:21]: Thanks for the info.

**Comment by atefBB** [comment]
@sagikazarmark @dbu @Nyholm @GrahamCampbell Thank you all!

**Comment by dbu** [comment]
thank you @atefBB for looking at the code and sending a pull request. don't let this discourage you and keep sending pull requests if you see things that you think can be improved! (before doing larger changes, best open an issue to discuss first, to avoid wasting time on the code)

**Pull request closed without merging**